packages/strapi-plugin-upload/admin/src/containers/HomePage/utils/generateStringFromParams.js

```diff
@@ -0,0 +1,33 @@
+import { isEmpty, toString } from 'lodash';
+
+const generateStringFromParams = query => {
+  let params = '';
+
+  Object.keys(query)
+    .filter(key => !isEmpty(toString(query[key])))
+    .forEach(key => {
+      const value = query[key];
+
+      if (key === 'filters') {
+        value.forEach(item => {
+          if (item.name.includes('mime') && item.value === 'file') {
```

Comment:
`if (item.name === 'mime' && item.value === 'file')`  you don't need the `includes` here.